events/events.go

```diff
@@ -0,0 +1,119 @@
+package events
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/reddit/baseplate.go/mqsend"
+
+	"github.com/apache/thrift/lib/go/thrift"
+)
+
+// Configuration values for the message queue.
+const (
+	// Max size in bytes for a single, serialized event.
+	MaxEventSize = 102400
+
+	// Max size of the events allowed in the message queue at one time.
+	MaxQueueSize = 10000
+
+	// Prefix added to the message queue name.
+	QueueNamePrefix = "events-"
+
+	// The default MaxPutTimeout to be used.
+	DefaultMaxPutTimeout = time.Millisecond * 50
+
+	// The default message queue name for v2 events.
+	DefaultV2Name = "v2"
+)
+
+// A Queue is an event queue.
+type Queue struct {
+	queue      mqsend.MessageQueue
+	maxTimeout time.Duration
+	pool       sync.Pool
```

Comment:
🔕 would it be better to call this `serializerPool`?

Reply:
Done.